webapp-admin/src/components/SystemUserForm.js

```diff
@@ -215,7 +224,7 @@ class SystemUserForm extends Component {
                         <label htmlFor="since">Since (UTC):
                             <div className="row">
                                 <div className="col-3">
-                                    <DatePicker selected={this.state.since} onChange={this.handleChangeSinceDate} />
+                                    <DatePicker selected={this.state.startDate} onChange={this.handleChangeSinceDate} />
```

Comment:
Since handleChangeSinceDate function is updating state.since the selected date will not be updated unless this works by being the last thing the user selected.